lldb/test/API/lang/swift/embedded/frame_variable/TestSwiftEmbeddedFrameVariable.py

```diff
@@ -31,3 +31,8 @@ def test(self):
         )
 
         # TODO: test enums when "rdar://119343683 (Embedded Swift trivial case enum fails to link)" is solved
+
+        self.expect("frame variable sup", substrs=["Sup) sup = ", "supField = 42"])
```

Comment:
If you find the time, it would be neat to replace these with more accurate check_variable() calls